README.rst

```diff
@@ -211,29 +211,46 @@ or on the SCons Discord server in
 Once you have discussed your issue on the users mailing list and the
 community has confirmed that it is either a new bug or a duplicate of an
 existing bug, then please follow the instructions the community provides
-to file a new bug or to add yourself to the CC list for an existing bug
+(including the issue template presented by GitHub)
+to file a new bug or to add yourself to the CC list for an existing bug.
 
 You can explore the list of existing bugs, which may include workarounds
 for the problem you've run into, on the
 `GitHub issue tracker <https://github.com/SCons/scons/issues>`_.
 
+Bug-fix Policy
+--------------
 
-Mailing Lists
-=============
+At this time, the application of bug-fix pull requests *normally* happens
+at the top of the main branch. In other words fixes are likely to appear
+in the next regular release and there won't be a bugfix update to a
+past release.  This is mainly down to (lack of) resources to maintain
```

Comment:
I'd drop the mainly due to. and the rest of the paragraph.. Just letting them know when to expect merged PR should suffice? the rest sounds apologetic..  Maybe add to the paragraph something like if you'd like to help the project move faster

Reply:
Sure, can shorten it.  Not sure someone popping in would make things move faster, but that's a different story.